src/components/app/DogFinder.js

```diff
@@ -1,58 +1,111 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Flex, Box } from '@rebass/grid';
-import Slider from '../core/Slider';
+import Select from '../core/Select';
 import Widget from '../core/Widget';
-import Dogs from './Dogs';
+import VisuallyHidden from '../core/VisuallyHidden';
+import LoadingContainer from '../core/LoadingContainer';
+import CardList from './CardList';
+import DogCard from './DogCard';
+import { STATUSES } from '../../consts';
 
-const SliderColumn = ({ children }) => (
-	<Box width={[1, 1 / 3]} px={3} py={[3, 0]}>
-		{children}
-	</Box>
+const FilterSelect = ({ name, options, value, disabled, onChange }) => (
+	<Select
+		fullWidth
+		disabled={disabled || options.length <= 1}
+		value={value}
+		onChange={event => {
+			const { value } = event.target;
+			const number = Number(value);
+			onChange(name, String(number) === value ? number : value);
+		}}
+	>
+		{options.map(option => (
+			<option key={option.value} value={option.value}>
+				{option.label}
+			</option>
+		))}
+	</Select>
 );
 
-const DogFinder = ({ filters, dogs, onFilterUpdate }) => (
+const DogFinder = ({
+	status,
+	filterOptions,
+	filters,
+	dogs,
+	onFilterUpdate,
+}) => (
 	<React.Fragment>
+		<VisuallyHidden>
+			<h3>Filters</h3>
```

Comment:
Maybe `placeholder`?

Reply:
What is `placeholder` here? Looks like I'm missing something ;-/